src/main/java/uk/gov/hmcts/dts/fact/mapit/MapitClient.java

```diff
@@ -12,4 +14,7 @@
 
     @GetMapping("${mapit.endpoint.postcode-search}/partial/{postcode}")
     MapitData getMapitDataWithPartial(@PathVariable("postcode") String postcode);
+
+    @GetMapping("${mapit.endpoint.area-search}/{area}?types=MTD,UTA,LBO,CTY")
+    Map<String, MapitArea> getMapitDataForLocalAuthorities(@PathVariable("area") String namePrefix);
```

Comment:
Why is the parameter called 'namePrefix'? Is that not for the entire local authority name?